docs/apis/migration-api-azure-container-and-queue.md

```diff
@@ -10,7 +10,7 @@ ms.localizationpriority: medium
 ---
 # SPO provided Migration Azure container and queue
 
-Microsoft’s Migration API requires the use of an Azure container for temporary storage. To simplify the process, you are now provided with a default container while using the migration API. If you choose, you can still provide your own Azure container.
+Microsoft’s Migration API requires the use of an Azure container for temporary storage. To simplify the process, you are now provided with a default container while using the migration API. To use the provided container you will need to [decorate your traffic correctly](https://docs.microsoft.com/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#how-to-decorate-your-http-traffic) to avoid throttling. If you choose, you can still provide your own Azure container.

```

Comment:
Please fix the build warning from the automated checks:

### [docs/apis/migration-api-azure-container-and-queue.md](https://github.com/SheaAhna/sp-dev-docs/blob/patch-1/docs/apis/migration-api-azure-container-and-queue.md)
 - **Line 13, Column 238**: **[Suggestion: docs-link-absolute - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#how-to-decorate-your-http-traffic' will be broken in isolated environments. Replace with a relative link.`````